src/App.tsx

```diff
@@ -1,48 +1,23 @@
-import { useState } from "react";
-import reactLogo from "./assets/react.svg";
-import viteLogo from "/vite.svg";
 import "./App.css";
+import "rsuite/dist/rsuite.min.css";
+import LandingPage from "./organisms/LandingPage/LandingPage";
 
 function App() {
-  const environment = import.meta.env.VITE_APP_ENV
-  const [count, setCount] = useState(0);
+  const environment = import.meta.env.VITE_APP_ENV;
   switch (environment) {
-    case 'dev':
-      document.title = 'Edudisha-Dev';
+    case "dev":
+      document.title = "Edudisha-Dev";
       break;
-    case 'stage':
-      document.title = 'Edudisha-Stage';
+    case "stage":
+      document.title = "Edudisha-Stage";
       break;
```

Comment:
can create one constant like 
const ENVIRONMENT  = {
           DEV: 'dev',
           STAGE: 'stage' .. and so on }
and use like that ENVIRONMENT.DEV .. AND so on

Reply:
not my code, added by someone else on main, had due to accepting both changes in conflicts its reflecting in my PR

Reply:
not my code, added by someone else on main, due to accepting both changes in conflicts its reflecting in my PR